ffi/examples/read-table/test_expr.c

```diff
@@ -0,0 +1,12 @@
+#include "delta_kernel_ffi.h"
```

Comment:
Can we actually just create a whole new `ffi/examples/visit-expression/visit_expr.c` and copy over whatever you need from the `read_table` example?